syntaxes/test/data/template.html

```diff
@@ -15,3 +15,22 @@
 <div [myProperty$]="val"></div>
 <div [%invalidProperty]="val"></div>
 <div [invalidProperty)="val"></div>
+
+<!-- Event binding test -->
+<button (click)="onClick($event)"></button>
+<input [ngModel]="model" (ngModelChange)="onModelChange($event)" />
```

Comment:
```suggestion
<input (ngModelChange)="onModelChange($event)" />
```

Reply:
I think you can even remove this test as the previous one covers the same idea.

Reply:
@ayazhafiz Resolved with 2c2bba9d09f7ea0c53c0d1d0ee9f15a3f6f76188